app/(candidate)/dashboard/text-messaging/p2p-setup/[step]/components/P2PSteps.js

```diff
@@ -0,0 +1,20 @@
+'use client'
+import Stepper from '@mui/material/Stepper'
+import Step from '@mui/material/Step'
+import StepLabel from '@mui/material/StepLabel'
+
+const steps = ['EIN', 'Website', 'Email', 'Review']
+
+export default function P2PSteps({ activeStep = 0 }) {
+  return (
+    <div className="max-w-3xl mx-auto">
+      <Stepper activeStep={activeStep} alternativeLabel>
+        {steps.map((label) => (
+          <Step key={label}>
+            <StepLabel>{label}</StepLabel>
+          </Step>
+        ))}
+      </Stepper>
+    </div>
+  )
+}
```

Comment:
These labels should be colocated w/ the `STEPS` constant in the `P2pSetupPage.js`, and fed into this stepper component.  Which, if we think about it, could easily be generalized to be just a generic `StyledStepper` component that can then be reused elsewhere:

```suggestion
export default function P2PSteps({ activeStep = 0, stepLabels }) {
  return (
    <div className="max-w-3xl mx-auto">
      <Stepper activeStep={activeStep} alternativeLabel>
        {stepLabels.map((label) => (
          <Step key={label}>
            <StepLabel>{label}</StepLabel>
          </Step>
        ))}
      </Stepper>
    </div>
  )
}
```